### PR TITLE
Kh 141622057 sorting foods

### PR DESCRIPTION
**Pull request by kheppenstall**
@Sh1pley covers the story for sorting foods by clicking the calories button in foods.html.

@case-eee @icorson3 Would love some feedback on keeping the counter to have the same button have different functionality (`var sortIndex`)

### CODE REVIEW

File: lib/foods.js [hunk 1]
```diff
@@ -99,9 +104,35 @@ function Food(name, calories) {
...
+function caloriesSort() {
+  $('#sort-calories').on('click', function() {
+    $('tr.food-row').hide();
+    sortFoods();
+    displayFoods();
+  })
+}
+
+function sortFoods() {
+  if (sortIndex % 3 === 0) { sortFoodsAscending(); }
```
Comment (Sh1pley) [inline lib/foods.js:122]: shnazzy. I like this.

File: lib/foods.js [hunk 2]
```diff
@@ -99,9 +104,35 @@ function Food(name, calories) {
...
+  else if (sortIndex % 3 === 1) { sortFoodsDescending(); }
+  else { setFoods(); }
+  sortIndex++;
+}
+
+function sortFoodsAscending() {
+  foods.sort(function(foodOne, foodTwo){
+    return foodTwo.calories - foodOne.calories
+  });
```
Comment (Sh1pley) [inline lib/foods.js:131]: Awesome

File: lib/foods.js [hunk 3]
```diff
@@ -99,9 +104,35 @@ function Food(name, calories) {
...
+}
+
+function sortFoodsDescending() {
+  foods.sort(function(foodOne, foodTwo){
+    return foodOne.calories - foodTwo.calories
+  });
```
Comment (Sh1pley) [inline lib/foods.js:137]: These are both very cool

**Review by Sh1pley — APPROVED** [review]

File: lib/foods.js (hunk 2 above)
Comment (case-eee) [reply lib/foods.js:131]: 👍

**Comment by case-eee** [comment]
this is looking pretty solid. 

@kheppenstall i'm not sure what your question is here actually - do you mind clarifying?

**Pull request merged**